core/services/relay/evm/evm.go

```diff
@@ -128,12 +149,27 @@ func NewRelayer(lggr logger.Logger, chain legacyevm.Chain, opts RelayerOpts) (*R
 		return nil, fmt.Errorf("cannot create evm relayer: %w", err)
 	}
 	lggr = lggr.Named("Relayer")
+	// run the migrations for the relayer
+	db, err := sqlx.Open(string(dialects.Postgres), opts.DBURL.String())
+	if err != nil {
+		return nil, fmt.Errorf("failed to open db: %w", err)
+	}
+
+	err = evmdb.Migrate(context.Background(), db, evmdb.Cfg{
+		Schema:  "evm_" + chain.ID().String(),
+		ChainID: ubig.New(chain.ID()),
+	})
```

Comment:
Have we thought through what happens if the node gets shut down while the relay is migrating? Is completing the migration what we want, or would it be better to abort?